autogen/oai/gemini.py

```diff
@@ -93,12 +103,28 @@ def create(self, params: Dict) -> ChatCompletion:
         messages = params.get("messages", [])
         stream = params.get("stream", False)
         n_response = params.get("n", 1)
-        params.get("temperature", 0.5)
-        params.get("top_p", 1.0)
-        params.get("max_tokens", 4096)
+
+
+        # Mapping, where Key is a term used by Autogen, and Value is a term used by Gemini
+        generation_config_params_mapping = {
+            # "n": "candidate_count", # Gemini supports only `n=1`
+            "stop_sequences": "stop_sequences",
+            "max_tokens": "max_output_tokens", 
+            "temperature": "temperature",
+            "top_p": "top_p",
+            "top_k": "top_k",
+
+            "max_output_tokens": "max_output_tokens"
+        }
```

Comment:
define as a constant out of the method.

Reply:
added as a class variable, it can now be accessed as GeminiClient.PARAMS_MAPPING
```
{'max_tokens': 'max_output_tokens',
 'stop_sequences': 'stop_sequences',
 'temperature': 'temperature',
 'top_p': 'top_p',
 'top_k': 'top_k',
 'max_output_tokens': 'max_output_tokens'}
 ```